core/src/main/java/com/sequenceiq/cloudbreak/repository/StackRepository.java

```diff
@@ -398,4 +399,36 @@
             "AND (s.type is not 'TEMPLATE' OR s.type is null) " +
             "AND ss.status not in (:notInStatuses)")
     List<JobResource> getJobResourcesNotIn(@Param("notInStatuses") Set<Status> notInStatuses);
+
+    @Query("SELECT s.id as id, s.resourceCrn as resourceCrn, s.name as name " +
+            "FROM Stack s " +
+            "WHERE s.terminated = null " +
+            "AND (s.type is not 'TEMPLATE' OR s.type is null) " +
+            "AND s.resourceCrn = :resourceCrn")
+    Optional<ResourceBasicView> findResourceBasicViewByResourceCrn(@Param("resourceCrn") String resourceCrn);
```

Comment:
Would it possibly make sense to add the new methods to the super-interface `WorkspaceResourceRepository` as well (in sync with how it is done in `AccountAwareResourceRepository`)?

Reply:
I think not, as I know we want to remove the Workspace, however if I found any reason I will add to as well.

Reply:
Got it.